src/traces/bar/plot.js

```diff
@@ -34,6 +34,10 @@ function getKeyFunc(trace) {
     }
 }
 
+function sign(v) {
+    return (v > 0) - (v < 0);
```

Comment:
Interesting sign function.
Please add a comment mentioning that it returns zero when both values are equal.

Reply:
I got it from [elsewhere in the codebase](https://github.com/plotly/plotly.js/blob/620a56bcc043e3391e6649bbab1b8597a8ca07e4/stackgl_modules/index.js#L2975), although I see now it's just ported from another library.